sdk/storage/azure-storage-blob/azure/storage/blob/_generated/models/_azure_blob_storage_enums.py

```diff
@@ -175,6 +175,7 @@ class ListContainersIncludeType(with_metaclass(_CaseInsensitiveEnumMeta, str, En
 
     METADATA = "metadata"
     DELETED = "deleted"
+    SYSTEM = "system"
```

Comment:
Is this supposed to be here?  This is an STG 80 feature.  If this class isn't public it doesn't matter.

Reply:
It is not public